### PR TITLE
Adding Makefile and rdk_gstreamer_utils_soc.cpp

### PR DESCRIPTION
**Pull request by sajanasadhan99**

### CODE REVIEW

File: rdk_gstreamer_utils_soc.cpp [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#include <gst/app/gstappsrc.h>
+#include "../rdk_gstreamer_utils.h"
+
+namespace rdk_gstreamer_utils
+{
+
+    enum rgu_audio_change_state {
```
Comment (deepthi-ps) [inline rdk_gstreamer_utils_soc.cpp:25]: I dont see this enum used anywhere. Please remove if not needed.

File: rdk_gstreamer_utils_soc.cpp [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        AUDCHG_INIT = 0,
+        AUDCHG_CMD = 1,
+        AUDCHG_SET = 2,
+        AUDCHG_ALIGN = 3,
+    };
+
+    GstElement *mCurAudioSink;
```
Comment (deepthi-ps) [inline rdk_gstreamer_utils_soc.cpp:32]: I dont see this element mCurAudioSink used anywhere. Please remove if not needed.

File: rdk_gstreamer_utils_soc.cpp [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    void setPixelAspectRatio_soc(GstCaps ** ppCaps,GstCaps *appsrcCaps,uint32_t pixelAspectRatioX,uint32_t pixelAspectRatioY)
+    {
+        return;
+    }
+
+    void deepElementAdded_soc (struct rdkGstreamerUtilsPlaybackGrp *pgstUtilsPlaybackGroup,GstBin* pipeline, GstBin* bin, GstElement* element)
+    {
+        return;
+    }
+    /**
```
Comment (deepthi-ps) [inline rdk_gstreamer_utils_soc.cpp:145]: Please remove this comment section

File: rdk_gstreamer_utils_soc.cpp (hunk 2 above)
Comment (sajanasadhan99) [reply rdk_gstreamer_utils_soc.cpp]: Done

File: rdk_gstreamer_utils_soc.cpp (hunk 3 above)
Comment (sajanasadhan99) [reply rdk_gstreamer_utils_soc.cpp]: Done

**Review by arun-madhavan-013 — APPROVED** [review]

**Pull request closed without merging**